lib/index.js

```diff
@@ -1,5 +1,9 @@
-export default (...args) => (req, res, next) => {
+export default (fallback, ...args) => (req, res, next) => {
   if (req.method === 'GET' && req.accepts('html')) {
-    res.sendFile(...args, err => err && next())
+    if (fallback instanceof Function) {
+      fallback(req, res, next)
+    } else {
+      res.sendFile(fallback, ...args, err => err && next())
```

Comment:
actually it doesn't break because of the rest parameter in line 0.

Check out [this example](https://babeljs.io/repl/#?evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-2&code=const%20test%20%3D%20(arg%2C%20...rest)%20%3D%3E%20console.log(arg%2C%20...rest)%0A%0Atest('one'%2C%20'two'%2C%20'three')%0A%0A%2F%2F%20does%20not%20equal%0A%0Aconsole.log('one'%2C%20%5B'two'%2C%20'three'%5D)%0A%0A%2F%2F%20but%20equals%0A%0Aconsole.log('one'%2C%20'two'%2C%20'three')) if it's unclear. 😄